packages/email-sender/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "pagopa-interop-email-sender",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Pagopa Interop Email Sender",
+  "main": "dist",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "scripts": {
+    "test": "vitest --config vitest.sql.config.ts",
```

Comment:
Can you call the config just `vitest.config.ts`? It's only SQL in this case, we don't need to specify it